docs/VisualStudioLinux.md

```diff
@@ -0,0 +1,132 @@
+# Using Visual Studio to Develop Enclave Applications for Linux
+
+This walkthrough assumes that you are using Visual Studio on a Windows development
+machine, and want to develop enclaves for Linux.  If you instead want to use Visual
+Studio Code on a development machine running either Windows or Linux, see the
+[VS Code instructions](https://github.com/microsoft/openenclave/blob/feature.new_platforms/new_platforms/vscode-extension/README.md).
+
+## Prerequisites
+
+To develop Linux applications using a Windows development machine, you will need the following.
+
+On the Windows development machine:
+
+- [Visual Studio](https://visualstudio.microsoft.com/downloads/) 2017 or 2019
+  (Community edition, or any other edition)
+- ["Linux development with C++" Visual Studio workload](https://devblogs.microsoft.com/cppblog/linux-development-with-c-in-visual-studio/),
+  installable via Tools -> Get Tools and Features (on VS2017) or Features (on VS2019) ->
+  Workloads -> Other Toolsets -> Linux Development with C++
+- [Open Enclave Wizard - Preview](https://marketplace.visualstudio.com/items?itemName=MS-TCPS.OpenEnclaveSDK-VSIX)
+  Visual Studio extension, v0.5 or later
+
+You will also need a build machine running Ubuntu 16.04 or Ubuntu 18.04.  This can be
+a remote Linux machine, but can simply be a "Generation 2" Linux VM on the Windows
+development machine configured as follows:
+
+1. Download an ISO for Ubuntu [18.04](http://releases.ubuntu.com/18.04/) or [16.04](http://releases.ubuntu.com/16.04/).
+   A "Server install image" is sufficient.
+1. Create a VM as follows.  Open "Hyper-V Manager", and do Action -> New -> Virtual Machine....  
+   - On the Specify Generation screen, choose Generation 2.
+   - On the Configure Networking screen, choose Default Switch to ensure you can connect to it with a debugger.
+   - On the Installation Options screen, choose the ISO file you downloaded.
+   - All other options can be either left as the defaults or changed as desired.
+1. Disable Secure Boot as follows.  In Hyper-V Manager, right click on the VM you created while it is stopped,
+  and select Settings... -> Security and uncheck Enable Secure Boot.
+1. Uncheck "Enable checkpoints" under the VM's Settings -> Checkpoints.
+1. Enable SGX for the VM as follows (this cannot be done from Hyper-V Manager):
+   - Download [VirtualMachineSgxSettings.psm1](https://raw.githubusercontent.com/microsoft/openenclave/f28cedce63be9673e20fe54563987189f2565637/new_platforms/scripts/VirtualMachineSgxSettings.psm1)
+   - Open an elevated PowerShell window (e.g., type "powershell" and click Run as Administrator)
```

Comment:
"(e.g., open the Start menu, type [...]"

Reply:
you don't have to "open the Start menu" to type, you type in the "Type here to search" box on Win10.  I'm going to assume they can find powershell just like I assume they can find Hyper-V manager, in any of several ways.